src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments/Infrastructure/OuterApi/Requests/DraftApprenticeship/CreatePriorLearningDataApimRequest.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace SFA.DAS.ProviderCommitments.Infrastructure.OuterApi.Requests.DraftApprenticeship
+{
+    public class CreatePriorLearningDataApimRequest : ApimSaveDataRequest
```

Comment:
Trivial, but why does this have an `ApimRequest` suffix, but `GetPriorLearningDataQueryRequest` does not?